### PR TITLE
Centos7 pg

### PR DESCRIPTION
**Pull request by jburel**
Handle the various versions of postgres

### CODE REVIEW

File: linux/step06_centos7_daemon.sh [hunk 1]
```diff
@@ -1,6 +1,15 @@
 #!/bin/bash
 
-cp omero-systemd.service /etc/systemd/system/omero.service
+PGVER=${PGVER:-pg94}
+
+if [ "$PGVER" = "pg95" ]; then
+	cp omero-pg95-systemd.service /etc/systemd/system/omero.service
+else
```
Comment (atarkowska) [inline linux/step06_centos7_daemon.sh]: @jburel could you make IF pg95 and IF pg94, we cannot have ELSE, otherwise my trick to skip PG won't work anymore

Comment (atarkowska) [inline linux/step06_centos7_daemon.sh]: exactly like https://github.com/ome/omero-install/blob/develop/linux/step01_centos7_pg_deps.sh

Comment (jburel) [inline linux/step06_centos7_daemon.sh]: true, forgot about that

**Comment by atarkowska** [comment]
To simplify testing I will cherry-pick your commits to https://github.com/ome/omero-install/pull/99

**Comment by joshmoore** [comment]
@aleksandra-tarkowska : I only see "Review config with new version of nginx." on 99. Is that what you meant?

**Comment by atarkowska** [comment]
@joshmoore I think you see that in #98 not #99

**Comment by jburel** [comment]
@aleksandra-tarkowska: added a new commit

**Comment by atarkowska** [comment]
thx merged to #99. this PR can be closed now

**Comment by joshmoore** [comment]
Thanks, @aleksandra-tarkowska. @jburel : I'll go ahead and proactively merge 99 if there are no objections to unblock both of you.

**Comment by atarkowska** [comment]
great, just hopefully travis will build green :)

**Pull request closed without merging**